core/offchain/src/api.rs

```diff
@@ -101,7 +103,7 @@ where
 	}
 
 	fn random_seed(&mut self) -> [u8; 32] {
-		unavailable_yet("random_seed")
+		self.rng.gen()
```

Comment:
I think [thread_rng](https://docs.rs/rand/0.7.2/rand/fn.thread_rng.html) should be enough.